report/src/index.ts

```diff
@@ -0,0 +1,361 @@
+            const w2 = new StyledTable()
+                .data([
+                    ["Cardinality", row.cardinality, "(~" + (row.cardinality / row.fill_count * 100).toFixed(0) + "%)"],
+                    ["Filled", row.fill_count, fillRate <= config.fillRateRedThreshold ? `(<b style="color:${config.redColor}">` + fillRate + "%</b>)" : "(" + fillRate + "%)"],
+                ])
+                .tbodyColumnStyles([
+                    { "font-weight": "bold", "font-size": config.secondaryFontSize + "px", "width": "1%" },
+                    { "font-weight": "normal", "font-size": config.secondaryFontSize + "px", "text-align": "right", "width": "auto" },
+                    { "font-weight": "normal", "font-size": config.secondaryFontSize + "px", "text-align": "left", "width": "1%" },
+                ])
+                ;
+            const grid = new Grid()
+                .gutter(0)
+                .surfaceShadow(false)
+                .surfacePadding("0")
+                .surfaceBorderWidth(0)
+                .setContent(0, 0, w1)
+                .setContent(1, 0, w2)
+                ;
+            return grid;
+        }
+        function getBreakdownWidget(row) {
+            if (row.cardinality_breakdown.Row.length > 0) {
+                return new BreakdownTable()
+                    .columns(["Cardinality", ""])
+                    .data(
+                        row.cardinality_breakdown.Row
+                            .map(row => [
+                                row.value.trim(),
+                                row.rec_count
+                            ])
+                    )
+                    .theadColumnStyles([
+                        {
+                            "text-align": "left",
+                            "font-size": config.secondaryFontSize + "px",
+                            "white-space": "nowrap"
+                        }
+                    ])
+                    .tbodyColumnStyles([
+                        {
+                            "max-width": "60px",
+                            "overflow": "hidden",
+                            "text-overflow": "ellipsis",
+                            "white-space": "nowrap",
+                        },
+                        {
+                            "text-align": "right",
+                            "font-size": config.secondaryFontSize + "px"
+                        }
+                    ])
+                    ;
+            } else {
+                return getNotAvailableWidget("Cardinality Breakdown", "N/A");
+            }
+        }
+        function getStatsWidget(row, dataWidth) {
+            if (row.is_numeric) {
+                return new StyledTable()
+                    .data([
+                        ["Mean", row.numeric_mean, ""],
+                        ["Std. Deviation", row.numeric_std_dev, ""],
+                        ["", "", ""],
+                        ["Quantiles", row.numeric_min, "Min"],
```

Comment:
Should this be `Quartiles`? We use "quartile" throughout the code, this is the only occurrence of "quantile".